src/agent/CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ target_link_libraries(Agent
     nlohmann_json::nlohmann_json
     fmt::fmt
     Logger
+    Config
```

Comment:
Why wasn't this needed before?

Reply:
Some other target used by the Agent may have linked against Config publicly.